### ISSUE TITLE
Direct payments from voucher

### ISSUE THREAD

**Issue by jamalv** [teamforus/general#882]
As a **sponsor** I would like to directly send money from an unassigned voucher to a provider so that I have more freedom while sending money.

Figma: [<https://www.figma.com/file/IntONFj90gliw0AkoH8ReL/?node-id=0%3A1>](https://www.figma.com/file/IntONFj90gliw0AkoH8ReL/?node-id=118%3A29484)

**Comment by jamalv** [comment]
Link to feature: https://forus.productboard.com/feature-board/planning/features/10414486

**Comment by jamalv** [comment]
Scope is about existing vouchers right now.

**Comment by jamalv** [comment]
@jamalv create separate issues for the 2 scopes

**Comment by lexlog** [comment]
intake notes - was blocked by BNG release
now its released and can be done

@jamalv will separate and then we will add it to sprint during sprint planning

**Comment by jamalv** [comment]
I've updated this issue and figma file to reflect the scope. Still need to create a new issue for the other scope(s)

**Comment by lexlog** [comment]
@dev-rminds @sashooa @ateamcms do we have any development already done? please update the status

refinement note - not refined, need to understand the status

**Comment by GerbenBosschieter** [comment]
Please take in consideration that these proposals do not meet the requirements for IIT.

**Comment by lexlog** [comment]
@ateamcms figma master is finally checked, we can start

**Comment by lexlog** [comment]
this issue is only for the first part
backend part will be done by @dev-rminds 

the second part of the issue - @jamalv will create separate

**Comment by lexlog** [comment]
refinement note - @ateamcms clear, ready to start in sprint

**Comment by lexlog** [comment]
refinement note - started, @ateamcms add the link for the PR pls

**Comment by ateamcms** [comment]
https://github.com/teamforus/forus-frontend/pull/2157

**Comment by lexlog** [comment]
refinement note - @dev-rminds review

**Comment by lexlog** [comment]
status update - backend part of this issue is in progress by @dev-rminds

**Comment by lexlog** [comment]
refinement note - @dev-rminds almost done backend part

**Comment by lexlog** [comment]
https://github.com/teamforus/forus-backend/pull/1451
